src/distPlus/distribution/Distributions.re

```diff
@@ -74,6 +74,21 @@ module Continuous = {
       (fn, {xyShape, interpolation}: t): option(DistTypes.continuousShape) =>
     fn(xyShape) |> E.O.fmap(make(interpolation));
 
+  let empty: DistTypes.continuousShape = {xyShape: XYShape.T.empty, interpolation: `Linear};
+  let combine =
+      (fn, t1: DistTypes.continuousShape, t2: DistTypes.continuousShape)
+      : DistTypes.continuousShape => {
+    make(`Linear, XYShape.Combine.combine(
+      ~xsSelection=ALL_XS,
+      ~xToYSelection=XYShape.XtoY.linear,
+      ~fn,
+      t1.xyShape,
+      t2.xyShape,
+    ));
+  };
+  let reduce = (fn, items) =>
+    items |> E.A.fold_left(combine(fn), empty);
```

Comment:
I like that you added this here. My intuition is that a fair bit more of the SymbolicDist functionality can be added to this file. For instance, discrete and mixed distributions can also have ``reduce`` functions.